scripts/sbom_scraper.sh

```diff
@@ -181,11 +215,17 @@ COMPONENT_NAME=${ORIG_COMPONENT_NAME%%:*}
 COMPONENT_VERSION=${ORIG_COMPONENT_NAME##*:}
```

Comment:
This is all docker image specific, move to conditional area above

Reply:
fixed